src/wallet/wallet.cpp

```diff
@@ -3297,6 +3297,47 @@ bool CWallet::CreateZerocoinSpendModel(string &stringError, string thirdPartyAdd
 
 }
 
+bool CWallet::CreateZerocoinSpendModel(CWalletTx& wtx, string &stringError, string thirdPartyAddress, vector<string> denomAmounts, bool forceUsed) {
+    if (!fFileBacked)
+        return false;
+     
+    vector<pair<int64_t, libzerocoin::CoinDenomination>> denominations;
+    for(vector<string>::iterator it = denomAmounts.begin(); it != denomAmounts.end(); it++){
+        string denomAmount = (*it).c_str();
```

Comment:
`const string& denomAmount = *it;` should be better to prevent instantiate another object.